packages/cli/src/util/telemetry/index.ts

```diff
@@ -156,7 +156,9 @@ export class TelemetryEventStore {
 
   save() {
     if (this.isDebug) {
-      this.output.debug(`${LogLabel} Flushing Events`);
+      // Intentionally not using `this.output.debug` as it will
+      // not write to stderr unless it is run with `--debug`
+      this.output.log(`${LogLabel} Flushing Events`);
       this.events.forEach(event => {
         this.output.debug(JSON.stringify(event));
```

Comment:
Shouldn't this be `this.output.log` as well?

Reply:
Yes! It was https://github.com/vercel/vercel/pull/12209/commits/64acfdd46b23fd98b88b4c0cd44e8d224becc753, I really don't know what happened there 🤔

Reply:
We probably also want to make the follow-up conditional an `else if` to keep in line with our plan to have debugging disable sending to server discussion